src/gbytes.f

```diff
@@ -76,43 +61,43 @@ SUBROUTINE GBYTES(IPACKD,IUNPKD,NOFF,NBITS,ISKIP,ITER)
 C
 C ISTEP IS THE DISTANCE IN BITS FROM THE START OF ONE BYTE TO THE NEXT.
 C
-      ISTEP  = NBITS + ISKIP      
+      ISTEP  = NBITS + ISKIP
 C
-C IWORDS TELLS HOW MANY WORDS TO SKIP FROM ONE BYTE TO THE NEXT.                
+C IWORDS TELLS HOW MANY WORDS TO SKIP FROM ONE BYTE TO THE NEXT.
 C
-      IWORDS = ISTEP / NBITSW    
+      IWORDS = ISTEP / NBITSW
 C
-C IBITS TELLS HOW MANY BITS TO SKIP AFTER SKIPPING IWORDS.                      
+C IBITS TELLS HOW MANY BITS TO SKIP AFTER SKIPPING IWORDS.
 C
-      IBITS  = MOD(ISTEP,NBITSW) 
+      IBITS  = MOD(ISTEP,NBITSW)
 C
       DO 10 I = 1,ITER
 C
-C MOVER SPECIFIES HOW FAR TO THE RIGHT A BYTE MUST BE MOVED IN ORDER            
+C MOVER SPECIFIES HOW FAR TO THE RIGHT A BYTE MUST BE MOVED IN ORDER
 C
-C    TO BE RIGHT ADJUSTED.                                                      
```

Comment:
Are you using some tool to remove whitespace at the end of the lines?

If so, in future, do a pass with just that change, and put up a PR, and then do the doxygen in a separate PR. That makes it easier to review.

Reply:
Do you want me to focus on all comments and not just the doxygen comments? Should they also be lowercase?

Reply:
Oh i see. I didn't use a tool, i'll put that back in.

Reply:
I'll keep an eye on it. I think that was the only file edited like that.

Reply:
For the doxygen work, just change the comment blocks at the top of the file and above each subprogram. Don't change the comments within the code itself.